packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -199,6 +202,7 @@ export default function MainFeedLayout({
     };
 
     const variables = getVariables();
+    const feedWithActions = isUpvoted || isSortableFeed;
```

Comment:
Not sure I get the naming, what is the intended difference here?

Reply:
The feeds where you have dropdown options for sorting.

Reply:
Hmm ok open to leave it alternative `feedWithActions` but also not much clearer

Reply:
Agree, it is a bit vague. Open for any suggestions 🚀

Reply:
No better suggestion, let's proceed with this.